### PR TITLE
Update showcase.json - add dragon eyes

### PR DESCRIPTION
**Pull request by vxyagr**
adding dragon eyes, a dice rolling game powered by ICP randomizer

Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

File: showcase.json [hunk 1]
```diff
@@ -3480,5 +3480,26 @@
   "twitter": "https://twitter.com/Mapzlive",
   "youtube": "",
   "submittableId": "43548962"
+  },
+  {
+  "name": "🐲 Dragon Eyes 👀",
+  "description": "Provably fair dice rolling game as a testament that randomness can be obtained on-chain",
+  "website": "https://icdragon.xyz/",
+  "logo": "/img/showcase/icdragon_logo.png",
```
Comment (jennifertrin) [inline showcase.json:3488]: Can you please upload a file with the same path?

File: showcase.json [hunk 2]
```diff
@@ -3480,5 +3480,26 @@
...
+  "screenshots": [
```
Comment (jennifertrin) [inline showcase.json:3489]: Can you please upload a file with the same path?

**Review by jennifertrin — CHANGES_REQUESTED** [review]

File: showcase.json (hunk 2 above)
Comment (vxyagr) [reply showcase.json:3489]: hi Jenn, 
yes, completed upload
please kindly continue

thanks

Comment (jennifertrin) [reply showcase.json:3489]: Thank you, @vxyagr for the update! Nice to see another project from the Loka team

**Review by jennifertrin — APPROVED** [review]

**Pull request merged**